index.js

```diff
@@ -74,6 +75,9 @@ function inject (dispatchFunc, options, callback) {
 function doInject (dispatchFunc, options, callback) {
   options = (typeof options === 'string' ? { url: options } : options)
 
+  let express = false;
+  ({ express, ...options } = options)
```

Comment:
This is nice to read, but it's slower than just reading a property.

Reply:
Just wanted to delete it from `options` as it's not needed further. Would you rather just keep it?

Reply:
please, otherwise it will reallocate options

Reply:
Aye, good point. Done.